NFT-Marketplace-Part-3.md

```diff
@@ -282,7 +282,7 @@ export default function Listing(props) {
   async function fetchNFTDetails() {
     try {
       // Get token URI from contract
-      let tokenURI = await ERC721Contract.tokenURI(0);
+      let tokenURI = await ERC721Contract.tokenURI(props.tokenId);
```

Comment:
This should be just `tokenId` not `props.tokenId`

Reply:
Oh no, my bad @karthik123karthik, you can't change that there because the `tokenId` is taken not by props but by `router.query` so in the URL.
So this change would be wrong.

Reply:
![image](https://user-images.githubusercontent.com/84716922/191049076-89f78c98-661a-4e92-84c6-0f1787f16b40.png)
why are we using `props.tokenId` here.